requirements-tests.txt

```diff
@@ -1,16 +1,9 @@
-# Type checkers and other linters that we test our stubs against. These should always
-# be pinned to a specific version to make failure reproducible. See also the
-# "tool.typeshed" section in pyproject.toml for additional type checkers.
-black==24.8.0            # must match .pre-commit-config.yaml
-flake8==7.1.1            # must match .pre-commit-config.yaml
-flake8-noqa==1.4.0       # must match .pre-commit-config.yaml
-flake8-pyi==24.6.0       # must match .pre-commit-config.yaml
```

Comment:
I just realized after approving, you can remove the `# must match requirements-tests.txt` text in `.pre-commit-config.yaml`